follower_default_send_mail_false/README.rst

```diff
@@ -0,0 +1,64 @@
+============================
+mail_send_mail_default_false
+============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fqrtl--custom-lightgray.png?logo=github
+    :target: https://github.com/OCA/qrtl-custom/tree/new_follower_not_sending_email_by_default/follower_default_send_mail_false
+    :alt: OCA/qrtl-custom
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/qrtl-custom-new_follower_not_sending_email_by_default/qrtl-custom-new_follower_not_sending_email_by_default-follower_default_send_mail_false
+    :alt: Translate me on Weblate
+
+|badge1| |badge2| |badge3| |badge4| 
+
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/qrtl-custom/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/qrtl-custom/issues/new?body=module:%20follower_default_send_mail_false%0Aversion:%20new_follower_not_sending_email_by_default%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Quartile Limited
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
```

Comment:
Please regenerate README, setting `--org-name=qrtl`. See relevant comments in https://qrtl.slab.com/posts/oca-maintainer-tools-r0ohrbgn#steps.